kickstarts/partials/post/ui_compile.ks.erb

```diff
@@ -1,5 +1,9 @@
 npm install -g yarn
 
+<% if File.exist?(@build_base.join(KS_PART_DIR, "post", "npm_registry.ks.erb")) %>
+  <%= render_partial "post/npm_registry" %>
```

Comment:
Shouldn't this be before the npm install up above?

Reply:
No, because I need `yarn` to set registry etc. (`yarn config set`).  That's how I had it first not realizing it and found that out by trying out...

Reply:
Edited the PR title and description. I will move this up in a follow up PR when I have updated npm_registry/cleanup ready.